src/utils.ts

```diff
@@ -129,16 +134,72 @@ export const assertEOSError = async (
 				error.json.error.name === eosErrorName,
 				`Expected ${eosErrorName}, got ${error.json.error.name} instead.`
 			);
-			return;
+			if (furtherErrorCheck) {
+				furtherErrorCheck(error)
+			};
 		} else {
 			// Fail if error not thrown by EOS
 			assert.fail(
 				`Expected EOS error ${eosErrorName}, but got ${JSON.stringify(error, null, 4)} instead.`
 			);
 		}
+		return true
```

Comment:
The missing semicolon here makes me think prettier hasn't run on this file. Could you please run `npm run format` to fix this up?

Also as an aside, we should have prettier set up as a precommit hook on the repo so this doesn't ever need to be done manually, but that's an enhancement.

Reply:
I have run and committed that as a separate commit because it changed quite a few files which I don't believe I had changed so it might not be running as expected on my machine.